packages/tenant-process/test/selfcareUpsertTenant.test.ts

```diff
@@ -12,16 +12,14 @@ import {
   TenantOnboardedV2,
   toTenantV2,
   CorrelationId,
+  PUBLIC_ADMINISTRATIONS_IDENTIFIER,
+  SCP,
 } from "pagopa-interop-models";
 import { describe, it, expect, vi, beforeAll, afterAll } from "vitest";
 import { genericLogger } from "pagopa-interop-commons";
 import { tenantApi } from "pagopa-interop-api-clients";
 import { getMockAuthData, getMockTenant } from "pagopa-interop-commons-test";
 import { selfcareIdConflict } from "../src/model/domain/errors.js";
-import {
-  PUBLIC_ADMINISTRATIONS_IDENTIFIER,
-  SCP,
-} from "../src/services/validators.js";
```

Comment:
Unrelated, this happened after porting main changes in delegation-process

Reply:
It should be imported from a `[consants.ts](https://github.com/pagopa/interop-be-monorepo/blob/ce363ee2444558e808d784dbe916ec1f08ac3c64/packages/models/src/constants.ts#L1)` file  from model package. Good catch 👍🏻